R/RevigoR-RDAVIDWebService.R

```diff
@@ -17,10 +17,12 @@ NULL
 #' @export
 #'
 #' @examples
+#' \dontrun{
```

Comment:
They don't work anymore? Why? It is not a good idea to include dontrun statements here

Reply:
Yeah, they didn't, but now they do, looks like it was a combination of X11 error and website error. Added `http_error` test.